src/database/config/db_build.sql

```diff
@@ -42,4 +42,7 @@ CREATE TABLE requirement (
 	prop_id INT REFERENCES proposal(id)
 );
 
-COMMIT ;
+INSERT INTO users(firstname, lastname, email, freelancer_url, photo_url, mobile_number, password)
+VALUES ('a', 'b', 'a.gmail.com', 'https://github.com/angham', 'https://www.iconspng.com/image/36709/face-avatar-man-male-handsome-3', '0097234567', '1234');
```

Comment:
@Angham116 You could use the hashed and salted version of 1234 as your password value that you store in the database here, which would let you type 1234 into the login field and successfully log in with this account.